src/pepr/operator/crd/validators/package-validator.spec.ts

```diff
@@ -471,6 +471,8 @@ describe("Test Allowed SSO Client Attributes", () => {
             "oauth2.device.authorization.grant.enabled": "true",
             "pkce.code.challenge.method": "S256",
             "client.session.idle.timeout": "3600",
+            "saml.client.signature": "false",
+            saml_assertion_consumer_url_post: "https://nexus.uds.dev/saml",
```

Comment:
commenting for posterity - this looks weird but @rjferguson21 mentioned that the quotes are removed by pepr format